mentors/responses.py

```diff
@@ -0,0 +1,20 @@
+get_mentors_200 ={
+  "message": "Mentors Fetched successfully.",
+  "data": [
+    {
+        "id": 0,
+        "name": "string",
+        "email": "string",
+        "contact": "string",
+        "detail": "string",
+        "description": "string",
+        "profile_pic_url": "string",
+        "year": 0,
+        "flag": True
+    }
+  ]
+}
+
+mentors_not_found_404 = {
+    "detail": "Mentors couldn't be fetched."

```

Comment:
Provide actual `keys`.  
```suggestion
    "message": "Mentors couldn't be fetched."

```